www/src/components/StickyResponsiveSidebar/StickyResponsiveSidebar.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import Container from 'components/Container';
+import {Component, React} from 'react';
+import isItemActive from 'utils/isItemActive';
+import Sidebar from 'templates/components/Sidebar';
+import {colors, media} from 'theme';
+import {Motion, spring} from 'react-motion';
+import ChevronSvg from './ChevronSvg';
+
+// TODO: memoize to save doing O(n) on the active section items + subitems every
+// time.
+function findActiveItemTitle(location, defaultActiveSection) {
+  const {items} = defaultActiveSection;
+  for (let i = 0, len = items.length; i < len; i++) {
+    const item = items[i];
+    if (isItemActive(location, item)) {
+      return item.title;
+    } else if (item.subitems && item.subitems.length) {
+      const {subitems} = item;
+      for (let j = 0, len2 = subitems.length; j < len2; j++) {
+        const subitem = subitems[j];
+        if (isItemActive(location, subitem)) {
+          return subitem.title;
+        }
+      }
+    }
+  }
+  // If nothing else is found, warn and default to section title
+  console.warn('No active item title found in <StickyResponsiveSidebar>');
+  return defaultActiveSection.title;
+}
+
+class StickyResponsiveSidebar extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      activeSection: props.defaultActiveSection,
+      open: false,
+    };
+    this.toggleOpen = this._toggleOpen.bind(this);
+  }
+
+  _toggleOpen() {
+    this.setState({open: !this.state.open});
+  }
+
+  render() {
+    const {defaultActiveSection, location} = this.props;
+    console.log('the defaultActiveSection is ;', defaultActiveSection);
+    console.log('this.props are ', this.props);
```

Comment:
These probably weren't intended to be left in ^

Reply:
Removed :)